ajax/libs/rxdb/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "rxdb",
+  "npmName": "rxdb",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
```

Comment:
Let's use `!(es|lib){,**/*}` which means we want to fetch all files under `basePath` other than the files under `es`/`lib`.